Added documentation for material mixing (#1511)
openmc-dev/openmc: docs/source/usersguide/materials.rst

             :attr:`Material.temperature` or :attr:`Cell.temperature`
             attributes, respectively.

-----------------
Material Mixtures
-----------------

In OpenMC it is possible to mix any number of materials to create a new material
with the correct nuclide composition and density. The 
:meth:`Material.mix_materials` method takes a list of materials and
a list of their mixing fractions. Mixing fractions can be provided as atomic 

> fractions, weight fractions or volume fractions. The fraction type

Reviewer: ```suggestion
fractions, weight fractions, or volume fractions. The fraction type
```

> is specifed by adding 'ao', 'wo' or 'vo' respectively as the third argument. 

Reviewer: ```suggestion
can be specified by passing 'ao', 'wo', or 'vo' as the third argument, respectively. 
```
